<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CardReaderWidget</class>
 <widget class="QWidget" name="CardReaderWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>239</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Card Reader</string>
  </property>
  <widget class="QPushButton" name="btnEnable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Включить терминал</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPayWithCard">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="text">
    <string>Оплатить картой</string>
   </property>
  </widget>
  <widget class="QLabel" name="sumTextLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма к оплате:</string>
   </property>
  </widget>
  <widget class="QLabel" name="sumLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
